packages/virtual-core/src/index.ts

```diff
@@ -261,6 +263,8 @@ export interface VirtualizerOptions<
   scrollingDelay?: number
   indexAttribute?: string
   initialMeasurementsCache?: VirtualItem[]
+  columns?: number
+  columnsSize?: number
```

Comment:
Not a big fan of having these as separate properties since they depend on each other and it means users you pass `columns` and not `columnsSize` or `columnsSize` and not `columns`.

An alternative would be `columns: number[]` where `number` would be the column's size. It would also make it possible to have columns with different sizes.

With that being said, having the column's size as a `number` closes the gate to variable columns size. For example you could have three columns, the one's `30%`, the second `30%` and the third `40%`. So another solution could be to have a `estimateColumnSize: (index: number) => number`.

Reply:
In the end the columns' size is entirely handled in user-land so we just need `columns: number`.